doc/index.rst

```diff
@@ -16,43 +16,60 @@ Mailadm keeps all configuration, token and user state in a single
-----------
+    To use mailadm, you need admin access to a `Mailcow
+    <https://mailcow.email/>`_ instance. You can run mailadm as a docker
+    container, either on the same machine as mailcow or somewhere else.
 
-Get a git copy of the mailadm repository and change into it.
+First get a git copy of the mailadm repository and change into it.
 
     $ git clone https://github.com/deltachat/mailadm
     $ cd mailadm
 
+Now you need to configure some environment variables in a file called ``.env``:
 
-Now **review and then run** this install script:
+* ``MAIL_DOMAIN``: the server name of the email addresses your users will have.
+* ``WEB_ENDPOINT``: the web endpoint of mailadm; make sure mailadm receives
+  POST requests at this address.
+* ``MAILCOW_ENDPOINT``: the API endpoint of your mailcow instance.
+* ``MAILCOW_TOKEN``: the access token for the mailcow API; you can generate it
+  in the mailcow admin interface.
 
-    $ sudo bash install_mailadm.sh
+In the end, your ``.env`` file will probably look like this:
 
-Note that you can call this command repetetively if you want
-to play around. By default this script:
+.. code:: bash
 
-- creates a `mailadm` user and install the mailadm software into it (from
-  the checkout, not via the Python package index)
+    export MAIL_DOMAIN=mail.example.org
+    export WEB_ENDPOINT=http://mailadm.example.org:3691/
+    export MAILCOW_ENDPOINT=https://mailcow-web.example.org/api/v1/
+    export MAILCOW_TOKEN=932848-324B2E-787E98-FCA29D-89789A
+    
+Now you can build and run the docker container:
 
-- creates config files at approprirate system locations
-  for integrating with systemd, dovecot, postfix and nginx.
+.. code:: bash
 
-- creates a `~mailadm/README.txt` file with further instructions
-  how to do 2-3 line changes to your existing nginx and dovecot
-  config files.
+    $ sudo docker build . -t mailadm-mailcow
+    $ touch mailadm.db
+    $ . .env && sudo docker run --mount type=bind,source=$PWD/mailadm.db,target=/mailadm.db --mount type=bind,source=$PWD/.env,target=/mailadm/.env --rm mailadm-mailcow /usr/bin/mailadm init --web-endpoint $WEB_ENDPOINT --mail-domain $MAIL_DOMAIN --mailcow-endpoint $MAILCOW_ENDPOINT --mailcow-token $MAILCOW_TOKEN
+    $ sudo docker run -d -p 3691:3691 --mount type=bind,source=$PWD/mailadm.db,target=/mailadm.db --name mailadm mailadm-mailcow gunicorn -b :3691 -w 1 mailadm.app:app
+
+.. note::
 
+    As the web endpoint also transmits passwords, it is highly recommended to
+    protect the WEB_ENDPOINT with HTTPS, for example through an nginx reverse
+    proxy. In this case, WEB_ENDPOINT needs to be the outward facing address,
+    in this example maybe something like
+    `https://mailadm.example.org/new_email/`.
+    
+    If you want to use the same TLS certificates as for mailcow, it might be
+    necessary to run them on the same machine or use some other mechanism to
+    synchronize them. Take this into account when you plan your production
+    setup.
```

Comment:
Not sure what you are warning about exactly.  The web-endpoint can run anywhere, and can drive mailcow instances remotely.   That the web endpoint runs on the same URL as the mailcow server is a special case and doesn't need to be discussed. I'd simply strike the paragraph.

Reply:
I struck it :)